movie_search/models.py

```diff
@@ -54,34 +46,121 @@ class Movie(models.Model):
     revenue = models.IntegerField(default=0, null=True, blank=True)
     homepage = models.CharField(max_length=200, null=True, blank=True)
     recommendation = models.ManyToManyField(
-        Recommendation, related_name="movies", blank=True
+        MovieRecommendation, related_name="movies", blank=True
     )
 
     def __str__(self):
-        return f"{self.movie_id}: {self.title}"
+        return f"{self.tmdb_id}: {self.title}"
+
+
+class MovieProvider(models.Model):
+    name = models.CharField(max_length=200, null=True, blank=True)
+    provider_id = models.PositiveSmallIntegerField(default=0)
+
+    def __str__(self):
+        return f"{self.provider_id}: {self.name}"
 
 
-class Video(models.Model):
+class MovieVideo(models.Model):
     movie = models.ForeignKey(Movie, on_delete=models.CASCADE, null=True, blank=True)
     name = models.CharField(max_length=200, null=True, blank=True)
     key = models.CharField(max_length=200, null=True, blank=True)
 
     def __str__(self):
-        return f"{self.movie.movie_id}: {self.name}, {self.key}"
+        return f"{self.movie.tmdb_id}: {self.name}, {self.key}"
 
     class Meta:
-        verbose_name_plural = "videos"
+        verbose_name_plural = "movie videos"
 
-class WatchList(models.Model):
+class MovieWatchList(models.Model):
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
     movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
 
     class Meta:
         unique_together = ('user', 'movie')
-        verbose_name_plural = 'watchlist'
+        verbose_name_plural = 'movie watchlist'
+
+    def __str__(self):
+        return f"{self.user.username} - {self.movie.title}"
+
+class TVSeriesGenre(models.Model):
```

Comment:
These genres are unique to tv series, we cannot leverage the existing genre model you already use for movies?

Reply:
We agreed on last call to separate them out, I believe.

Reply:
Because they were different right?